### DISCUSSION TITLE
Customize input[type="date"]

### DISCUSSION THREAD

**Discussion by Yuniro** [tailwindlabs/tailwindcss#6160] [Help]
### What version of @tailwindcss/forms are you using?

v0.3.4

### What version of Node.js are you using?

v14.15.4

### What browser are you using?

Chrome

### What operating system are you using?

MacOS

### Reproduction repository

none

### Describe your issue

I want to customize input[type="date"] tag.
- change date format
-  replace calendar icon, 
- remove slash between day, month and year

could you kindly let me know how can I customize?

Thank you

**Comment by buncis** [comment]
for the date format some say its impossible check this https://github.com/tailwindlabs/tailwindcss-forms/issues/15 and this https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/date#value and this https://stackoverflow.com/questions/7372038/is-there-any-way-to-change-input-type-date-format

it's possible if you use javascript though
